tools/build.py

```diff
@@ -64,8 +64,7 @@ def init_options():
                     argv.append('--%s=%s' % (opt_key, val))
 
     # Apply command line argument to argv.
-    argv = argv + ([arg for arg in sys.argv[1:]
-                    if not arg.startswith('--config=')])
+    argv = argv + sys.argv[1:]
```

Comment:
Could you check that it is safe to remove the filtering ? The original author of the line is @chokobole.